example/src/App.tsx

```diff
@@ -586,6 +622,23 @@ export default function App() {
       }
     }
 
+    if (message.text.startsWith('/tts ')) {
+      const text = message.text.slice(5)
+      const prompt = await context.getFormattedAudioCompletion(null, text)
+      const guideTokens = await context.getAudioCompletionGuideTokens(text)
+      const { audio_tokens: audioTokens } = await context.completion({
+        temperature: 0.1,
+        penalty_repeat: 1.1,
+        prompt,
+        guide_tokens: guideTokens,
+      }, () => {})
+      if (audioTokens) {
+        const audio = await context.decodeAudioTokens(audioTokens)
+        console.log('audio length', audio.length / 24000) // WavTokenizer's sample rate is 24000
+      }
+      return
+    }
+
```

Comment:
Can you make a message bubble, it can press and play the audio, may be using [react-native-audio-api](https://www.npmjs.com/package/react-native-audio-api)?